node_modules/col-assets/lib/api.js

```diff
@@ -144,6 +147,14 @@ var getAsset = module.exports.getAsset = function(ctx, id, opts, callback) {
       'required': false
     });
   }
+  if (opts.includeWhiteboardUsages) {
```

Comment:
Thanks @nicolaasmatthijs. The asset_whiteboard_elements table for exported whiteboards doesn't currently have a column for the asset id of the element (as opposed to asset id of the whiteboard). It's included in the JSON but not in a way that's queryable. Probably the quickest way to make an association possible is to add a column?